package.json

```diff
@@ -39,8 +39,5 @@
     "mocha": "^9.1.2",
     "mocha-cakes-2": "^3.3.0",
     "supertest": "^6.1.6"
-  },
```

Comment:
This was introduced in node v14.17.0, could you:

1. update the required node version to 14.17
2. bump the major version, since this will be a breaking change (which effects our internal site-platform)
3. update CHANGELOG.md